scripts/zones/Caedarva_Mire/TextIDs.lua

```diff
@@ -22,5 +22,5 @@ MEMBER_TOO_FAR = 7468; -- One or more party members are too far away from the en
 NOTHING_OUT_OF_ORDINARY = 7377; --There is nothing out of the ordinary here.
         NOTHING_HAPPENS = 7445; -- Nothing happens...
 
- JAZARAATS_HEADSTONE = 7520; -- The name ir Jazaraat is engraved on the headstone... 
+ JAZARAATS_HEADSTONE = 7520; -- The name ir Jazaraatis engraved on the headstone...
```

Comment:
From the filtered-dialog-table-079.xml it wrote:
```
<thing type="DialogTableEntry"><field name="index">7520</field><field name="text">The name ir Jazaraatis engraved on the headstone... </field></thing>
```

I think it has something to do with the curly quotes around the name ir Jazaraat. I noticed the batch job wouldn't pick up on another curly quote (trading the waterskin I think) until both the offending quote _and_ the space after it were removed. To be honest, I have no clue how the tool works to grab fragments when the string is full of offensive characters. I just give it whatever will stop the FIXMEs. :sweat_smile: